pkg/kubelet/images/image_manager_test.go

```diff
@@ -153,20 +155,32 @@ func pullerTestCases() []pullerTestCase {
 			qps:        2000.0,
 			burst:      0,
 			expected: []pullerExpects{
-				{[]string{"GetImageRef"}, ErrImagePull},
-				{[]string{"GetImageRef"}, ErrImagePull},
-				{[]string{"GetImageRef"}, ErrImagePullBackOff},
+				{[]string{"GetImageRef"}, ErrImagePull, true, false},
+				{[]string{"GetImageRef"}, ErrImagePull, true, false},
+				{[]string{"GetImageRef"}, ErrImagePullBackOff, false, false},
 			}},
 	}
 }
 
-type mockPodPullingTimeRecorder struct{}
+type mockPodPullingTimeRecorder struct {
+	startedPullingRecorded  bool
+	finishedPullingRecorded bool
+}
+
+func (m *mockPodPullingTimeRecorder) RecordImageStartedPulling(podUID types.UID) {
+	m.startedPullingRecorded = true
+}
 
-func (m *mockPodPullingTimeRecorder) RecordImageStartedPulling(podUID types.UID) {}
+func (m *mockPodPullingTimeRecorder) RecordImageFinishedPulling(podUID types.UID) {
+	m.finishedPullingRecorded = true
+}
 
-func (m *mockPodPullingTimeRecorder) RecordImageFinishedPulling(podUID types.UID) {}
+func (m *mockPodPullingTimeRecorder) Reset() {
```

Comment:
```suggestion
func (m *mockPodPullingTimeRecorder) reset() {
```

we don't need to export it, do we?